File: src/main/java/com/github/frcsty/frozenactions/actions/player/MessageAction.java
package com.github.frcsty.frozenactions.actions.player;

import com.github.frcsty.frozenactions.actions.Action;
import com.github.frcsty.frozenactions.actions.ActionContext;
import com.github.frcsty.frozenactions.util.Extensions;
import org.jetbrains.annotations.NotNull;

public final class MessageAction implements Action {

    @Override
    public @NotNull String getId() {
        return "MESSAGE";
    }

    @Override
    public void run(final @NotNull ActionContext context) {
        Extensions.sendTranslatedMessage(context.getPlayer(), context.getData());
    }
}

File: src/main/kotlin/com/github/frcsty/frozenactions/actions/player/MessageAction.kt
package com.github.frcsty.frozenactions.actions.player

import com.github.frcsty.frozenactions.actions.Action
import com.github.frcsty.frozenactions.util.sendTranslatedMessage
import org.bukkit.entity.Player

object MessageAction : Action {
    override val id = "MESSAGE"

    override fun run(player: Player, data: String) = player.sendTranslatedMessage(data)
}